plugins/icons_suruplus/change_color.sh

```diff
@@ -166,12 +166,13 @@ if [ "$SURUPLUS_GRADIENT_ENABLED" = "true" ] && [ -n "${SURUPLUS_GRADIENT1:-}" ]
 	echo ":: Replacing gradient colors..."
 	find "$tmp_dir"/Suru++/apps/16 \
 		"$tmp_dir"/Suru++/devices/16 \
+		"$tmp_dir"/Suru++/mimetypes/16 \
 		"$tmp_dir"/Suru++/places/16 \
 		-type f -name '*.svg' \
 		-exec sed -i'' \
-			-e 's/currentColor/url(#arrongin)/g' \
-			-e "s/dd9b44/$SURUPLUS_GRADIENT1/g" \
-			-e "s/ad6c16/$SURUPLUS_GRADIENT2/g" '{}' \;
+			-e 's/currentColor/url(#oomox)/g' \
+			-e "s/efefe7/$SURUPLUS_GRADIENT1/g" \
+			-e "s/8f8f8b/$SURUPLUS_GRADIENT2/g" '{}' \;
```

Comment:
i still didn't got why this part is needed

Reply:
because `arrongin` is for Suru++ Arrongin icons theme. The `oomox` is reserved for Oomox. 

If you think it is unnecessary, you can ignore it and pull only translations.